pkg/userop/client.go

```diff
@@ -198,15 +213,16 @@ func NewClient(config ClientConfig) (Client, error) {
 		return nil, fmt.Errorf("failed to build initCode middleware: %w", err)
 	}
 
-	getGasLimits, err := getGasLimitsMiddleware(bundlerRPC, config)
+	getGasLimits, err := getGasLimitsMiddleware(bundlerRPC, pmRPC, config)
 	if err != nil {
 		return nil, fmt.Errorf("failed to build gas limits estimation middleware: %w", err)
 	}
 
 	return &backend{
-		provider: providerRPC,
-		bundler:  bundlerRPC,
-		chainID:  chainID,
+		provider:  providerRPC,
+		bundler:   bundlerRPC,
+		pmBackend: pmRPC,
```

Comment:
I'd rename this since the contraction pm is not immediately obvious for people who have less experience with userops. But it's up to debate
```suggestion
		paymaster: paymasterRPC,
```

Reply:
f53a477c00c669e62d72418df74059acc9d9320c